SensorFrame.java

```diff
@@ -0,0 +1,86 @@
+package com.company;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import javax.swing.*;
+import javax.swing.border.TitledBorder;
+
+import static java.awt.BorderLayout.CENTER;
+import static java.awt.BorderLayout.PAGE_END;
+
+public class SensorFrame extends JFrame {
+    public SensorFrame() {
+        setTitle("Sensor Tracker");
+        setLayout(new GridLayout(3,1));
+
+        JPanel  temperaturePnl = new JPanel();
+        temperaturePnl.setBorder(new TitledBorder("Temperature"));
+        add(temperaturePnl);
+        temperaturePnl.setLayout(new BorderLayout());
+
+        SensorInterface temp = new TemperatureAdaptor();
+        Double tvalue = temp.read();
+        String tstatus = temp.status();
+        JLabel tempTxt = new JLabel(tstatus + " --> " + tvalue);
+        temperaturePnl.add(tempTxt, PAGE_END);
+        JPanel tBar = new JPanel();
+        if(tstatus == "OK") {
+            tBar.setBackground(Color.GREEN);
+        } else if(tstatus == "Critical") {
+            tBar.setBackground(Color.YELLOW);
+        } else {
+            tBar.setBackground(Color.RED);
+        }
+        temperaturePnl.add(tBar, CENTER);
+
+        JPanel  pressurePnl = new JPanel();
+        pressurePnl.setBorder(new TitledBorder("Pressure"));
+        add(pressurePnl);
+        pressurePnl.setLayout(new BorderLayout());
+
+        SensorInterface pressure = new PressureAdaptor();
+        Double pvalue = pressure.read();

```

Comment:
I'd recommend writing out pressure in your variable names, if just for the business-world possibility that this could be a program with many more p-sensors